curated_transformers/layers/feedforward.py

```diff
@@ -25,15 +25,23 @@ class PointwiseFeedForward(Module):
 
     This layer is applied pointwise, meaning that the same
     transformation is applied to each sequence element. This
-    transformation is: :math:`g(xW_1 + b_1)W_2 + b_2`, where :math:`W_1`` and :math:`b_1`
-    transform the input to an intermediate width, :math:`g` is a non-linear activation
-    function and :math:`W_2` and :math:`b_2` transform the output of the activation back to
-    the input width.
+    transformation is:
+
+    .. math::
+        g(xW_1 + b_1)W_2 + b_2
+
+    :math:`W_1` and :math:`b_1` transform the input to an
+    intermediate width, :math:`g` is a non-linear activation
+    function and :math:`W_2` and :math:`b_2` transform the
+    output of the activation back to the input width.
 
     Gated Linear Units (`Dauphin et al., 2016`_; `Shazeer, 2020`_) are also
-    supported. Gating applies the transformation
-    :math:`(g(xW_g + b_g) * (xW_1 + b_1))W_2 + b_2`, where :math:`W_g` and :math:`b_g`
-    are the affine transformations for the gate.
+    supported. Gating applies the following transformation:
+
+    .. math::
+        (g(xW_g + b_g) * (xW_1 + b_1))W_2 + b_2
+
+    :math:`W_g` and :math:`b_g` are the affine transformations for the gate.
```

Comment:
```suggestion
    :math:`W_g` and :math:`b_g` are the affine transformation for the gate.
```

(Also wrong in the original doc.)